doc/classes/EditorExportPlugin.xml

```diff
@@ -44,6 +46,7 @@
 			<param index="1" name="path" type="String" />
 			<description>
 				Customize a scene. If changes are made to it, return the same or a new scene. Otherwise, return [code]null[/code]. If a new scene is returned, it is up to you to dispose of the old one.
+                Calling [method skip] inside this callback will make the file not included in the export.
```

Comment:
This should be indented with tabs (same below).

Reply:
Thanks! I used a different editor this time and didn't notice its behaviour. I'll update this PR to fix those...